clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -760,18 +760,43 @@ public FetchManagerMetrics(Metrics metrics, String metricGrpPrefix) {
         }
 
         public void recordTopicFetchMetrics(String topic, int bytes, int records) {
+            Map<String, String> metricTags = new HashMap<>();
+            metricTags.put("topic", "topic-" + topic.replace(".", "_"));
```

Comment:
@hachikuji Are you ok with removing it, as tag-key is 'topic' in this case and it will be part of metric path?